centreon/tests/e2e/features/Additional-connectors/02-ACC-listing/index.ts

```diff
@@ -0,0 +1,98 @@
+    method: 'GET',
+    url: '/centreon/api/latest/configuration/additional-connector-configurations?*'
+  }).as('getConnectorPage');
+  cy.intercept({
+    method: 'POST',
+    url: '/centreon/api/latest/configuration/additional-connector-configurations'
+  }).as('addAdditionalConnector');
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/latest/configuration/additional-connector-configurations/*'
+  }).as('getConnectorDetail');
+});
+
+after(() => {
+  cy.stopContainers();
+});
+
+Given('a non-admin user is logged in', () => {
+  cy.loginByTypeOfUser({
+    jsonName: 'user-non-admin-for-ACC',
+    loginViaApi: false
+  });
+});
+
+When('the user clicks on the Specific Connector Configuration page', () => {
+  cy.navigateTo({
+    page: 'Additional Connector Configurations',
+    rootItemNumber: 0
+  });
+});
+
+Then('the user sees the Specific Connector Configuration page', () => {
+  cy.wait('@getConnectorPage');
+});
+
+Then('there is no additional connector configuration listed', () => {
+  cy.get('*[role="rowgroup"]')
+    .should('contain', 'No result found');
+});
+
+Given('a non-admin user is in the Specific Connector Configuration page', () => {
+  cy.loginByTypeOfUser({
+    jsonName: 'user-non-admin-for-ACC',
+    loginViaApi: false
+  });
+  cy.visit('/centreon/configuration/additional-connector-configurations');
+  cy.wait('@getConnectorPage');
+});
+
+Given('an already existing additional connector configuration', () => {
+  cy.getByLabel({ label: 'Add', tag: 'button' }).click();
+  cy.getByLabel({ label: 'Name', tag: 'input' }).type('Connector-001');
+  cy.get('#mui-component-select-type').should('have.text', 'VMWare 6/7');
+  cy.getByLabel({ label: 'Select poller(s)', tag: 'input' }).click();
+  cy.contains('Central').click();
+  cy.get('[id="vCenternamevalue"]').clear().type('vCenter-001');
+  cy.get('[id="URLvalue"]').clear().type('https://10.0.0.0/sdk');
+  cy.get('[id="Usernamevalue"]').type('admin');
+  cy.get('[id="Passwordvalue"]').type('Abcde!2021');
+  cy.get('[id="Portvalue"]').should('have.value', '5700');
+  cy.getByLabel({ label: 'Create', tag: 'button' }).click();
+  cy.wait('@addAdditionalConnector');
+  cy.get('*[role="rowgroup"]')
+    .should('contain', 'Connector-001');
+});
+
+When('the user clicks on the Edit button of the additional connector configuration', () => {
+  cy.getByLabel({ label: 'Edit connector configuration', tag: 'button' }).click();
+});
+
+Then('a pop up is displayed with all of the additional connector informations', () => {
+  cy.wait('@getConnectorDetail');
+  cy.contains('Update additional connector configuration').should('be.visible');
+  cy.getByLabel({ label: 'Name', tag: 'input' }).should('have.value', 'Connector-001');
+  cy.getByLabel({ label: 'Description', tag: 'textarea' }).should('be.empty');
+  cy.get('#mui-component-select-type').should('have.text', 'VMWare 6/7');
+  cy.get('*[class="MuiChip-label MuiChip-labelMedium css-14vsv3w"]').should('contain', 'Central');
```

Comment:
MuiChip-label MuiChip-labelMedium css-14vsv3w is dynamic due to the inclusion of the class css-14vsv3w, which can change with different builds or styles, leading to potential failures in our tests.
cy.get('*[class="MuiChip-label MuiChip-labelMedium"]').should('contain', 'Central');